options.rb

```diff
@@ -15,38 +15,9 @@ def options(app)
     number = gets.chomp.to_i
 
```

Comment:
**[OPTIONAL]**

- [ ] It would be nice if you use the same picture on your Twitter account which you have used on LinkedIn because many recruiters are on Twitter and when they will visit your Twitter account then they should get a professional vibe from your account and the first thing they are gonna notice is your Twitter account dp :thinking: so now you know what you should do :+1: